src/main/java/org/opencds/cqf/config/STU3LibraryLoader.java

```diff
@@ -87,18 +87,16 @@ public Library toElmLibrary(org.hl7.fhir.dstu3.model.Library library) {
     }
 
     private Library loadLibrary(VersionedIdentifier libraryIdentifier) {
-        //IdType id = new IdType(libraryIdentifier.getId());
-        //org.hl7.fhir.dstu3.model.Library library = provider.getDao().read(id);
-
+        IdType id = new IdType(libraryIdentifier.getId().replaceAll("_", "-"));
```

Comment:
I just removed the commented out stuff and saved the libraryIdentifier.getId() call into an object because it was being called twice. I also added some logic that replaces underscores with hyphens because I was trying to test a QDM measure that uses underscores in the library identifier. Adding that little bit of code was easier than changing the CQL and all that goes with that. I can confirm that my edits don't conflict with library name resolution. The name resolution happens in another class, that I did not edit.